tests/Unit/Numerical/LinearAlgebra/Test_Determinant.cpp

```diff
@@ -0,0 +1,151 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/DataVector.hpp"
+#include "Numerical/LinearAlgebra/Determinant.hpp"
+
+TEST_CASE("Unit.Numerical.LinearAlgebra.Determinant",
+          "[LinearAlgebra][Numerical][Unit]") {
+  // Catch's Approx has a default tolerance of 100*FLOAT_EPS ~ 10^{-5}.
```

Comment:
I'm not sure we will want this comment in every single test. I think this is something we should only have in the (eventual) documentation for the testing framework. If you feel like it you could add it to the GroupDefs.hpp file, if not it can be done later. (This is completely optional so please don't feel like you need to do this :) )